task/coverity-availability-check/0.2/coverity-availability-check.yaml

```diff
@@ -1,88 +1,88 @@
 apiVersion: tekton.dev/v1
 kind: Task
 metadata:
+  annotations:
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/tags: konflux
   labels:
     app.kubernetes.io/version: "0.1"
-  annotations:
-    tekton.dev/pipelines.minVersion: "0.12.1"
-    tekton.dev/tags: "konflux"
-  name: coverity-availability-check
+  name: coverity-availability-check-oci-ta
```

Comment:
Hm. Shouldn't this be _without_ the `oci-ta`?

Reply:
Oof, good catch! The output of kustomize went into the symlinked file. Fixed